packages/hardhat-core/src/internal/core/errors-list.ts

```diff
@@ -855,6 +855,16 @@ Use a relative import instead of referencing the package's name.`,
 Please double check your imports and installed libraries.`,
       shouldBeReported: false,
     },
+    INVALID_IMPORT_OF_DIRECTORY: {
+      number: 414,
+      message:
+        "Invalid import %imported% from %from%. Attempting to import a directory. Directories cannot be imported.",
+      title: "Invalid import: a directory cannot be imported",
+      description: `A Solidity file is attempting to import a directory, which is not possible.
+
+      Please double check your imports.`,
```

Comment:
Text inside backticks is used verbatim and interpreted as markdown, so this shouldn't be indented. In practice I think this doesn't change anything because whatever markdown pre-processor we are using right now doesn't seem to handle [indented code blocks](https://spec.commonmark.org/0.30/#indented-code-blocks), but that could easily change.

```suggestion
Please double check your imports.`,
```